ansible/roles/cartodb.postgresql/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+ - name: Add cartodb postgresql apt repository
+   apt_repository: repo=ppa:cartodb/postgresql-9.3 update_cache=yes
+
+ - name: Install postgresql client packages
+   apt: name={{ item }} state=present
+   with_items:
+    - libpq5
+    - libpq-dev
+    - postgresql-client-{{ postgresql_version }}
+    - postgresql-client-common
+
+ - name: Install postgresql server packages
+   apt: name={{ item }} state=present
+   with_items:
+    - postgresql-{{ postgresql_version }}
+    - postgresql-contrib-{{ postgresql_version }}
+    - postgresql-server-dev-{{ postgresql_version }}
+    - postgresql-plpython-{{ postgresql_version }}
+
+ - name: Add CartoDB pg-schema-trigger repo
+   apt_repository: repo=ppa:cartodb/pg-schema-trigger state=present update_cache=yes
+
+ - name: Install postgresql schema triggers
+   apt: name=postgresql-9.3-pg-schema-triggers state=present
+
+ - name: Configure PostgreSQL host-based authentication
+   template: src=pg_hba.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/pg_hba.conf
+   notify:
+     - Restart PostgreSQL
+
+ - name: Add cartodb publicuser
+   command: createuser publicuser --no-createrole --no-createdb --no-superuser
+   sudo: True
+   sudo_user: postgres
+   # TODO: Come up with better way to handle user already created
```

Comment:
Sort of. I punted because the helpful `azavea.postgresql-support` role, which installs the dependencies for that module, has packages that conflict with the cartodb postgresql ppa. I figured it was something we could create an issue for and look at later, since the error message this throws (and skips) when the user already exists is very clear in the ansible log and I had more pressing things to try to figure out as part of this first iteration.